front/lib/resources/group_resource.ts

```diff
@@ -177,6 +181,51 @@ export class GroupResource extends BaseResource<GroupModel> {
     return new this(GroupModel, group.get());
   }
 
+  static async addMember(
```

Comment:
It would probably makes more sense to have it as a class method than a static method.

Reply:
Yes 🤦🏻‍♀️